src/3rd_party/VersionApi.h

```diff
@@ -14,7 +14,8 @@
 #define _WIN32_WINNT_WIN7           0x0601
 #define _WIN32_WINNT_WIN8           0x0602
 #define _WIN32_WINNT_WINBLUE        0x0603
-#define _WIN32_WINNT_WIN10          0x0A00        
+#define _WIN32_WINNT_WIN10          0x0A00  
+#define _WIN32_WINNT_WIN11          0x55F0
```

Comment:
Incorrect for build 22000 >=
For 22000>=. Major is 10, minor is 0

Reply:
the win 11 check works just fine like it is I haven't encountered any issues when only checking if build is over 22000 since anything over 22000 is win 11 but hey correct me if I'm wrong. (especially since I think ms will release some new win10 build soon)